tikv/kv.go

```diff
@@ -595,16 +612,11 @@ func (s *KVStore) updateSafeTS(ctx context.Context) {
 
 			var (
 				safeTS uint64
-				err    error
 			)
-			storeIDStr := strconv.Itoa(int(storeID))
-			// Try to get the minimum resolved timestamp of the store from PD.
 			if s.pdHttpClient != nil {
-				safeTS, err = s.pdHttpClient.GetStoreMinResolvedTS(ctx, storeID)
-				if err != nil {
-					logutil.BgLogger().Debug("get resolved TS from PD failed", zap.Error(err), zap.Uint64("store-id", storeID))
-				}
+				safeTS = StoreMinResolvedTSs[storeID]
 			}
+			storeIDStr := strconv.FormatUint(storeID, 10)
```

Comment:
What about moving this into the `if` block below?

Reply:
- metrics need string type to report... so need to at if block outer
- StoreMinResolvedTSs need unit type, so we cannot put it in func's params :(